recipes/r-tftargets/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = 'tftargets' %}
+{% set version = '1.3' %}
+
+package:
+  name: r-{{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - https://github.com/slowkow/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: e27ed65c283779b4dcda6e0704a37112a81bfedb198b3d7d036c1a80083670ca
+
+build:
+  number: 1
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+
```

Comment:
Remove new line.

Reply:
Thanks for the thorough feedback and suggestions! It is much appreciated. I'll be sure to take a closer look into each of the config params next time around.

Reply:
No worries, that is why we require 4-eyes review :)